src/stores/Annotation/Annotation.js

```diff
@@ -1137,6 +1153,14 @@ export const Annotation = types
           };
 
           area = createArea(areaSnapshot);
+
+          // store copy of the original result inside the area
+          // useful when you need to serialize a result without
+          // updating it from current/actual data
+          // For safety reasons this object is always readonly
+          Object.defineProperty(area, '_rawResult', {
```

Comment:
I'd wrap this in FF just in case. This may lead to data corruption if something will go wrong